e2e/e2e_test.go

```diff
@@ -113,9 +118,9 @@ func Test_No_Ethereum_TX_Submission(t *testing.T) {
 
 	memo := fmt.Sprintf("%s-0-0", receiverAddress)
 
-	// Step 1 - Verify the transfer of Hbars to the Bridge Account
-	transactionResponse, _ := verifyTransferToBridgeAccount(setupEnv, memo, whbarReceiverAddress, t)
-
+	// Step 1 - Verify the transfer of HTS Token to the Bridge Account
```

Comment:
This test should be left to work with hbars. We can change the name to `Test_HBAR_No_Ethereum_TX_Submission` as-well